kong/cli/utils/signal.lua

```diff
@@ -62,8 +64,16 @@ local NGINX_SEARCH_PATHS = {
 -- Try to find an `nginx` executable in defined paths, or in $PATH
 -- @return Path to found executable or nil if none was found
 local function find_nginx()
-  for i = 1, #NGINX_SEARCH_PATHS + 1 do
-    local prefix = NGINX_SEARCH_PATHS[i] and NGINX_SEARCH_PATHS[i] or ""
```

Comment:
Because there is an issue where the `nginx` executable is in a different folder specified into the `PATH` environment variable, and we are not checking that. If you read the line below, now this will check in both our paths and the `PATH` environment variable.